rule.go

```diff
@@ -914,3 +911,14 @@ func (r *Rule) HasVar(s string) bool {
 	}
 	return false
 }
+
+//getSidMgs returns a string representing a sidmsg.map entry
```

Comment:
Comments should always have a space following `//` and end in a full stop `.`

Reply:
done.